README.rst

```diff
@@ -1013,6 +1013,17 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_position`` (``'always_first'``, ``'always_last'``, ``'default'``): forces the
```

Comment:
Yeah I know, I'm in fav. of moving away from that. Configuring fixers typically means you want them to do something ;) 
Besides it is also a performance impact as the fixer would still be constructed, configured and tested for candidates, something the fixer is even run as well (like here, the candidate check could see the fixer is configured to do nothing and return `false`)
Throwing the misconfigured exception would be better UX/DX IMHO, however for the current ones it could be a BC break depending on your view of feature vs. bug.